build/config_builder

```diff
@@ -75,6 +81,7 @@ foreach($generatedConfig as $stability => $ignored) {
 
 // Daily updates
 $maxMajor = (int) max(array_keys($majorVersions));
+$now = date('Y-m');
```

Comment:
Just to be sure, but i think it is fine: if EOL is 2024-04 it means the final release will be in that month.

Reply:
Yes (we can have a little delay since we need to build the `config.php` to have the EOL updated in it)